README.md

```diff
@@ -223,3 +223,18 @@ This will produce tests as follows:
 Provisioning mechanisms can be provided externally by implementing `io.kroxylicious.testing.kafka.api.KafkaClusterProvisioningStrategy` and declaring it as a [Java service](https://www.baeldung.com/java-spi) (e.g. in a  `META-INF/services/io.kroxylicious.testing.kafka.api.KafkaClusterProvisioningStrategy` file that is available on the test-time classpath).
 
 You can also provide your own constraint annotation types by annotating them with the `@io.kroxylicious.testing.kafka.api.KafkaClusterConstraint` meta-annotation. Such custom constraint annotations will only be understood by your custom provisioning strategy, so the in-JVM and testcontainers-based clusters provided by this project then can't be used.
+
+## Compilation with format validator
+
+As the project should follow the same format for all the files, in case a format error is detected when running the `ci` profile:
```

Comment:
The maven build defaults to fixing formatting errors. The `ci` profile is purely intended so that the build fails if the formatting is wrong so that a human validates the fixes.

Reply:
Agree with that. But it happens to me that when testing locally without `ci` profile everything seems to be fine, but on github action that uses `ci` profile, the build fails because of that. I had to do it manually following the steps I put in the README file.

This is a suggestion just in case someone has the same issue than me.

Reply:
Hmmm, that suggests something else is going wrong for you locally. One shouldn't need to run those commands `mvn install` should fix it up anyway.

I'm happy to keep the hints there but would like to work out why we don't just do the right thing by default.